packages/react/src/Suspense.en.mdx

```diff
@@ -5,9 +5,9 @@ title: Suspense
 
 This component is just wrapping [React's Suspense](https://reactjs.org/docs/react-api.html#reactsuspense). to use Suspense easily in Server-side rendering environment like [Next.js](https://nextjs.org)
 
-### Default mode
```

Comment:
I just want to make there is no mode, just Suspense, Suspense.CSROnly component.
I want to remove new concept to learn for users